packages/envied/lib/src/envied_base.dart

```diff
@@ -86,6 +86,10 @@ final class Envied {
   /// Escapes single quotes and newlines in the value.
   final bool rawStrings;
 
+  /// A seed can be provided if the obfuscation randomness needs to remain
+  /// reproducible across builds.
```

Comment:
```suggestion
  /// A seed can be provided if the obfuscation randomness needs to remain
  /// reproducible across builds.
  /// **Note**: This will make the `Random` instance non-secure!
```